corehq/apps/domain/models.py

```diff
@@ -1227,3 +1230,39 @@ def as_dict(self):
             'deactivate_url': self.deactivate_url(),
             'activate_url': self.activate_url(),
         }
+
+
+class DomainAuditRecordEntry(models.Model):
+    domain = models.TextField(unique=True, db_index=True)
+    cp_n_downloads_custom_exports = models.BigIntegerField(default=0)
```

Comment:
Not a blocker, but I think it would be reasonable to use IntegerField for these columns, since I doubt they will ever go over two billion

https://docs.djangoproject.com/en/1.11/ref/models/fields/#integerfield